docker/apply/views/help/terms-and-conditions.html

```diff
@@ -5,150 +5,190 @@
+    </p>
+    <p class="govuk-body">Read our <a href="/help/cookies" class="govuk-link">cookies policy</a>.</p>
+
+    <h2 class="govuk-heading-l">Using ‘Find a professional service abroad'</h2>
+    <div class="govuk-body">
+      <p>
+        The Foreign, Commonwealth & Development Office (FCDO) provides lists of service providers for information only,
+        to assist British nationals who may need support overseas. These lists are not exhaustive and are subject to
+        change at any time.
+      </p>
+      <p>
+        None of the service providers are endorsed or recommended by the FCDO and the FCDO does not assume any
+        responsibility, including legal responsibility or liability, to any person who uses or relies upon these service
+        providers or this information.
       </p>
-      <p class="govuk-body">
-        Read our <a href="/help/cookies" class="govuk-link">cookies policy</a>.
+      <p>
+        You should research whether a service provider will be suitable for you, considering your specific
+        circumstances.
       </p>
+      <p>
+        The FCDO does not accept any liability for any loss or damage suffered by a person, legal person or company, as
+        a result of using these service providers or this information.
+      </p>
+      <p>
+        Complaints about any organisations on the list should be directed to the nearest consulate, embassy or high
+        commission.
+      </p>
+    </div>
 
-      <h2 class="govuk-heading-l">Using ‘Find a professional service abroad'</h2>
-      <div class="govuk-body">
-        <p>
-          The Foreign, Commonwealth & Development Office (FCDO) provides lists of service providers for information only,
-          to assist British nationals who may need support overseas. These lists are not exhaustive and are subject to change at any time.
-        </p>
-        <p>
-          None of the service providers are endorsed or recommended by the FCDO and the FCDO does not assume any
-          responsibility, including legal responsibility or liability, to any person who uses or relies upon these
-          service providers or this information.
-        </p>
-        <p>
-          You should research whether a service provider will be suitable for you, considering your specific circumstances.
-        </p>
-        <p>
-          The FCDO does not accept any liability for any loss or damage suffered by a person, legal person or company,
-          as a result of using these service providers or this information.
-        </p>
-        <p>
-          Complaints about any organisations on the list should be directed to the nearest consulate, embassy or high commission.
-        </p>
-      </div>
+    <h2 class="govuk-heading-l">Applying to be added to ‘Find a professional service abroad’</h2>
+    <div class="govuk-body">
+      <p>
+        Inclusion on any of the 'Find a professional service' service is not an endorsement by the FCDO and you are not
```

Comment:
should be 
Inclusion on any of the  the 'Find a a professional service lawyer abroad' services

Reply:
Good spot thanks. Ha, it looks like **lawyer** has been crossed out. Don't know if that is a recent change 😁 

<img width="1038" alt="Screenshot 2022-09-20 at 17 50 48" src="https://user-images.githubusercontent.com/1377253/191317952-66a01cd3-63fa-474b-af98-4aad5141bcd7.png">

Reply:
To answer your other question:
> also, can you check that the prettier config is set to 120 chars

It looks like the setting is there in prettier, but I don't think my prettier is set to work on nunjucks files 🤔 

I can look into it

https://github.com/UKForeignOffice/lists/blob/chore/update-t-and-cs/.prettierrc.json#L2

Reply:
ah looks like I gave a typo in my suggestion where it includes "lawyer".  Anyway the latest change looks good.  Thanks for confirming on prettier - I'll approve once verified as I'd like to preserve the 120 limit on njk files as well

Reply:
Hey Ali, I'ave added a prettier override for .njk files but it didn't seem to work for me in VSCode :( Maybe it might work in WebStorm 🤷

I had to rename the .njk file to .html to apply the formatting, then revert back to .njk